services/schema-registry-service/src/main/resources/application.yml

```diff
@@ -17,6 +17,14 @@ management:
     info:
       enabled: true
 
+miranum:
```

Comment:
security is not implemented in schema-registry-service yet. Should work without that as well.